styles/config.css

```diff
@@ -189,12 +189,18 @@ body {
 
   .main-content {
     margin-left: 0;
-    padding-left: 16px;
-    padding-right: 16px;
+    padding-left: 0px;
+    padding-right: 0px;
+    width: 90%;

```

Comment:
@karinisantanna just curious why give 90% width over constant side padding? Did you find any issue in the original setup?

Reply:
@cTxplorer I had the impression that the paddings were breaking the block, so i thought about reducing the width and using margin 0 auto to center it. 
But I checked it again and I think it was a problem with the mobile emulator I was using!

I revert this part! Can you validate please?